java/ql/src/semmle/code/java/Statement.qll

```diff
@@ -423,6 +423,14 @@ class SwitchCase extends Stmt, @case {
    */
   SwitchExpr getSwitchExpr() { result.getACase() = this }
 
+  /**
+   * Gets the expression of the surrounding switch that this case is compared
+   * against.
+   */
+  Expr getTestExpr() {
```

Comment:
Could we rename this to `getSelectorExpr()` for consistency with the terminology introduced in the [Preview Language Specification for JEP 325](http://cr.openjdk.java.net/~gbierman/switch-expressions.html#jep325-15.28)? Similarly for the variable names elsewhere in the diff, e.g. `switchTest` -> `switchSelector`.